test/pom.xml

```diff
@@ -242,7 +242,8 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>matrix-auth</artifactId>
-      <version>3.2.5</version>
+      <!-- TODO https://github.com/jenkinsci/matrix-auth-plugin/pull/169 -->
```

Comment:
https://github.com/jenkinsci/matrix-auth-plugin/pull/169

Reply:
Seems prudent to run this through a `bom` build then.

Reply:
https://github.com/jenkinsci/bom/pull/4743 looks good

Reply:
https://github.com/jenkinsci/bom/pull/4743#issuecomment-2743163024

Reply:
Really good now.